common/tsconfig.json

```diff
@@ -24,7 +24,7 @@
   },
   "include": [
     "src"
-  ],
+, "../talentsearch/resources/js/tests/localize.test.tsx"  ],
```

Comment:
Is there another way?  We don't want to have common depend on anything from talentsearch, right?  I'm thinking of issue #898.

Reply:
This was done in PR #1781 as well.  I don't love it TBH.

Reply:
I think this might have been a mistake on my part? I'm just gonna try taking it out.

Reply:
Yep, still works. Don't know why that ended up there 😄

Reply:
You're right, seems fine without it.  🤷‍♂️